app/src/main/java/org/jamienicol/episodes/EpisodesListFragment.java

```diff
@@ -194,6 +194,14 @@ public void bindView(View view, Context context, Cursor cursor) {
 					DateFormat.getDateInstance().format(date);
 				dateView.setText(dateText);
 				dateView.setVisibility(View.VISIBLE);
+
+				// grey out episode name if upcoming episode
+				if (date.after(new Date())) {
+					nameView.setTextColor(context.getResources().getColor(android.R.color.secondary_text_dark));
```

Comment:
hmm. Shouldn't we be using a "light" colour, since it's on a light background? But secondary_text_light looks like the same colour as primary_text_light, at least on stock lollipop.

tertiary_text_light, on the other hand, looks like exactly what we want. Could you change it to use that instead, as long as that also looks good on your device?